docs/capabilities.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Solaar Capabilities
+layout: page
+---
+# Solaar capabilities
+
+[**Solaar**][solaar] reports on and controls [Logitech][logitech] devices (keyboards, mice, and trackballs) that connect to your computer via a Logitech USB receiver (a very small piece of hardware that plugs into one of your USB ports) and communicate with the receiver using Logitech's HID++ protocol.  Solaar is designed to detect all devices paired with your receivers, and at the very least display some basic information about them.
+
+At this moment, all [Unifying][unifying] receivers are supported (devices with
+USB ID `046d:c52b` or `046d:c532`) as are several Lightspeed Receivers, but only some newer Nano receivers (devices with USB ID `046d:c52f`, and `046d:c534`) are supported. Note that Linux incorrectly states that some Nano receivers are Unifying receivers.
```

Comment:
In the list at `devices.md`, there are 12 Nano's (only according to that list one supports paring with 2 devices (i.e. c534), the others with 1 device). 

"as are several Lightspeed Receivers, but only some newer Nano receivers (devices with USB ID `046d:c52f`, and `046d:c534`) are supported" =>
"as are several Lightspeed receivers and a dozen newer Nano receivers are supported.

Reply:
Good catch.  I changed the sentence to:
At this moment, all [Unifying][unifying] receivers are supported (devices with
USB ID `046d:c52b` or `046d:c532`) as are several Lightspeed Receivers and a dozen Nano receivers.  
I also took out the crack about Linux misidentifying Nano receivers as Unifying receivers.
It would be useful to get confirmation from Logitech about just which receivers are which kind.  I only have the c534 and c52b.